azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/MessageSender.java

```diff
@@ -893,12 +889,13 @@ public void run() {
                     public void onEvent() {
```

Comment:
>onEvent [](start = 32, length = 7)

this change is same as messagereceiver change.. marking here.. to be sure...

Reply:
Same comment as above.